src/server.rs

```diff
@@ -60,9 +60,16 @@ async fn create_server() -> axum::Server<AddrIncoming, IntoMakeService<axum::Rou
     server
 }
 
-fn setup_logging() {
+fn setup_logging(args: &cli::Args) {
+    let mut trace_level = tracing::Level::INFO;
+    if args.verbose {
+        trace_level = tracing::Level::DEBUG;
+    }
+    if args.vv {
+        trace_level = tracing::Level::TRACE;
+    }
     tracing_subscriber::fmt()
-        .with_max_level(tracing::Level::INFO) // TODO: --verbose should use DEBUG
+        .with_max_level(trace_level)
```

Comment:
You should also set the Axum logging level here,

https://github.com/juspay/nix-browser/blob/132ed807fa909b0eaa73871b28f0f6c255b50703/src/server.rs#L53-L54

Reply:
(Make sure to merge from `main` first, because I touched these lines today)